docs/the-guide/chapter-7-track-model-evolutions-in-the-cicd-pipeline-with-cml/.gitlab-ci.yml

```diff
@@ -7,7 +7,7 @@ variables:
   # only cache local items.
   PIP_CACHE_DIR: "$CI_PROJECT_DIR/.cache/pip"
```

Comment:
I guess we can remove this line as we don't use pip anymore?

Reply:
Pip is still used under the hood by pipx.